Directory.Build.props

```diff
@@ -64,10 +64,10 @@
     <TargetGroup Condition="'$(TargetGroup)' == ''">netcoreapp</TargetGroup>
     <OSGroup Condition="'$(OSGroup)' == ''">$(DefaultOSGroup)</OSGroup>
     <ConfigurationGroup Condition="'$(ConfigurationGroup)' == ''">Debug</ConfigurationGroup>
-    <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture)</HostArch>
-    <ArchGroup Condition="'$(ArchGroup)' == '' AND '$(HostArch)' == 'Arm'">arm</ArchGroup>
-    <ArchGroup Condition="'$(ArchGroup)' == '' AND '$(HostArch)' == 'Arm64'">arm64</ArchGroup>
-    <ArchGroup Condition="'$(ArchGroup)' == ''">x64</ArchGroup>
+    <HostArch>$([System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture.ToString().ToLowerInvariant())</HostArch>
+    <!-- MSBuild inside Visual Studio is always x86. Force it to x64. -->
+    <HostArch Condition="'$(BuildingInsideVisualStudio)' == 'true'">x64</HostArch>
+    <ArchGroup Condition="'$(ArchGroup)' == ''">$(HostArch)</ArchGroup>
```

Comment:
This might still break some scenarios and will require a bunch of testing. Today we only use HostArch in case of arm or arm64, but use x64 in all other cases. This would effectively change that.